src/Jackett.Common/Indexers/Milkie.cs

```diff
@@ -0,0 +1,167 @@
+                queryParams.Add("query", query.SearchTerm);
+            }
+
+            if (query.HasSpecifiedCategories)
+            {
+                queryParams.Add("categories", string.Join(",", MapTorznabCapsToTrackers(query)));
+            }
+
+            var endpoint = TorrentsEndpoint + "?" + queryParams.GetQueryString();
+            var headers = new Dictionary<string, string>() { { "x-milkie-auth", configData.Key.Value } };
+            var jsonResponse = await RequestStringWithCookies(endpoint, null, null, headers);
+
+            var releases = new List<ReleaseInfo>();
+
+            try
+            {
+                var response = JsonConvert.DeserializeObject<MilkieResponse>(jsonResponse.Content);
+
+                var dlQueryParams = new NameValueCollection
+                {
+                    { "key", configData.Key.Value }
+                };
+
+                foreach (var torrent in response.Torrents)
+                {
+                    var link = new Uri($"{TorrentsEndpoint}/{torrent.Id}/torrent?{dlQueryParams.GetQueryString()}");
+                    var comments = new Uri($"{SiteLink}browse/{torrent.Id}");
+
+                    var release = new ReleaseInfo()
+                    {
+                        Title = torrent.ReleaseName,
+                        Link = link,
+                        Comments = comments,
+                        Guid = comments,
+                        Size = torrent.Size,
+                        Category = MapTrackerCatToNewznab(torrent.Category.ToString()),
+                        Seeders = torrent.Seeders,
+                        Peers = torrent.Seeders + torrent.PartialSeeders + torrent.Leechers,
+                        Grabs = torrent.Downloaded,
+                        UploadVolumeFactor = 1,
+                        DownloadVolumeFactor = 0,
+                        MinimumRatio = 1,
+                        MinimumSeedTime = 172800,
+                        PublishDate = DateTimeUtil.FromUnknown(torrent.CreatedAt)
+                    };
+
+                    releases.Add(release);
+                }
+            }
+            catch(Exception ex)
+            {
+                OnParseError(jsonResponse.Content, ex);
+            }
+
+            return releases;
+        }
+
+        private class MilkieResponse
+        {
+            public int Hits { get; set; }
+            public int Took { get; set; }
+            public MilkieTorrent[] Torrents { get; set; }
+        }
+
+        private class MilkieTorrent
+        {
+            public string Id { get; set; }
+            public string ReleaseName { get; set; }
+            public int Category { get; set; }
+            public int Downloaded { get; set; }
+            public int Seeders { get; set; }
+            public int PartialSeeders { get; set; }
+            public int Leechers { get; set; }
+            public long Size { get; set; }
+            public string CreatedAt { get; set; }
```

Comment:
@ngosang I believe there are attributes or other modifiers so we can deserialize this into DateTime directly. So when we do convert trackers we should keep that in mind. (not blocking)